.github/workflows/kind_integration.yml

```diff
@@ -65,6 +68,33 @@ jobs:
         name: image-archives
         path: /home/runner/archives
   # todo: Keep in sync with `release.yml`
+  windows_static_cli_tests:
+    name: Static CLI tests (windows)
+    runs-on: windows-latest
+    needs: [docker_build]
+    steps:
+    - name: Checkout code
+      # actions/checkout@v2
+      uses: actions/checkout@722adc6
+    - name: Try to load cached Go modules
+      # actions/cache@v1.1.2
+      uses: actions/cache@70655ec
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-
+    - name: Download image archives
+      # actions/download-artifact@v1
+      uses: actions/download-artifact@18f0f59
+      with:
+        name: image-archives
+    - name: Install CLI
+      run: |
+        cp image-archives\cli\windows\linkerd $HOME\linkerd.exe
```

Comment:
Nit: I think we can save ourselves this step by feeding `image-archives\cli\windows\linkerd` directly into `--linkerd` below (IMO the smaller these build files are, the better :wink: )

Reply:
Hmm, The main reason for doing this was to add `.exe` without which windows acts weirdly. Should we just rename it?

Reply:
Oh I see. In that case you can leave it as is :+1: